web/tests/typeahead_helper.test.js

```diff
@@ -455,7 +455,7 @@ test("sort_recipients all mention", () => {
         current_topic: "Linux topic",
     });
 
-    assertSameEmails(results, [all_obj, a_bot, a_user, b_user_1, b_user_2, b_user_3, b_bot, zman]);
```

Comment:
These last couple changes are because for users with no dm history at all, we prefer human to bot users in `compare_by_pms`